algorithmic_efficiency/workloads/fastmri/workload.py

```diff
@@ -34,11 +34,11 @@ def num_eval_train_examples(self):
 
   @property
   def num_validation_examples(self):
-    return 7135
+    return 3554
```

Comment:
I think we haven't discussed yet how we should handle the case where `(num_{eval_train, validation, test}_examples % global_batch_size) % N_GPU != 0`.

Reply:
yeah, we will need to support a padded final batch, which I think we sort of do right now, but we need to make sure if consistent. the batches all already have a 'weights' key that we can use to mask out any padding. I'll open an issue to make sure we don't forget.